3-With-Intune MAM/IntuneClient/active-directory-xamarin-intune.Android/IntuneMAMConnector.cs

```diff
@@ -19,26 +22,34 @@ public class IntuneMAMConnector : IIntuneMAMConnector
         /// <returns></returns>
         public async Task DoMAMRegisterAsync(IntuneAppProtectionPolicyRequiredException exProtection)
         {
+            Log.Info(GetType().Name, "***Begin DoMAMRegisterAsync");
+            Debug.WriteLine("***Begin DoMAMRegisterAsync");
             // reset the registered event
             PCAWrapper.MAMRegsiteredEvent.Reset();
 
             // Invoke compliance API on a different thread
-            await Task.Run(() =>
-                                {
-                                    IMAMComplianceManager mgr = MAMComponents.Get<IMAMComplianceManager>();
-                                    mgr.RemediateCompliance(exProtection.Upn, exProtection.AccountUserId, exProtection.TenantId, exProtection.AuthorityUrl, false);
-                                }).ConfigureAwait(false);
+            await Task.Run(() =>
+            {
+                Log.Info(GetType().Name, "Attempting to remediateCompliance.");

```

Comment:
nit: have a log method in the intune connector class.

Reply:
It is just for logging if required. Do not need to have a method for logging.